worklets/resources/referrer-tests.js

```diff
@@ -188,7 +188,7 @@ function runReferrerTests(workletType) {
                              scriptOrigins: { topLevel: 'remote',
                                               descendant: 'same' } });
   }, 'Importing a same-origin script from a remote-origin worklet script ' +
```

Comment:
This is ambiguous. Is it same-origin with the remote-origin script? I guess not, but to me it initially reads that way.

Reply:
I meant for every `{same,cross}-origin` statement to be relative to the document, but that might be confusing. Could make this more "absolute" with something like "Importing a script on origin-a from a script on origin-b". Would that be better? Or even more explicit: "From origin-a, fetch a script on origin-b. The origin-b script imports a descendant from on origin-a"

Reply:
I don't really have a good suggestion and it was already a problem.